select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -465,4 +465,12 @@ public void onComponentTagBody(final MarkupStream markupStream, final ComponentT
         }
 		replaceComponentTagBody(markupStream, openTag, buffer);
 	}
+	
+	/**
+	 * Empty input is acceptable
+	 */
+	@Override
+	public boolean isInputNullable() {
+		return true;
+	}
```

Comment:
But that is pretty natural for this type of component. So I recommend to have that by default and override if other behavior is needed: honestly, I don't see reasons for other behavior in real life:)